apps/pigment-css-next-app/src/app/material-ui/react-grid/page.tsx

```diff
@@ -1,3 +1,4 @@
+'use client';
```

Comment:
I am asking for a check from @mui/pigment-css team.
Other pages seem to have this declaration.
After bumping `next` these two pages started failing the build. 🙈